src/Modal/CreateEventModal.tsx

```diff
@@ -19,10 +22,19 @@ const CreateEventModal = ({
   const handleCloseModal = () => {
     closeModal(false);
   };
-  const handleSaveEvent = (event: Event) => {
-    console.log('Event was saved :', event);
-    closeModal(false);
+  const handleSaveEvent = async (event: Event) => {
+    try {
+      const savedEvent = await saveEventToServer(event);
+      const updatedEvents = [...events, savedEvent];
+      setEvents(updatedEvents);
+      
+      console.log('Event was saved:', savedEvent);
```

Comment:
remove console.logs

Reply:
does the console.error stay in the "catch" block?

Reply:
In general we do not want to have any console logs in production code and the error should be handled somehow, so that user would know that smth is not ok, what to do next. In this project scope it can stay.